webrtc.html

```diff
@@ -11497,18 +11517,56 @@ <h2>
                     <p>
                       If either (but not both) of <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}}
                       and <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}} are [= map/exist | missing =],
-                      or if they both [=map/exist=] and <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}}
-                      is different from <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}}, return
-                      <code>false</code>.
+                      return <code>false</code>.
                     </p>
                   </li>
+                  <li>
+                    <p>
+                      If both <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}} and
+                      <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}} [=map/exist=], and are in key/value format, run the following
+                      steps:
+                    </p>
+                    <ol>
+                      <li>
+                        <p>
+                          Let <var>firstMediaFormat</var> be a key-value map of the media formats constructed
+                          from <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}} and <var>secondMediaFormat</var>
+                          be a key-value map of the media formats constructed from
+                          <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}}.
+                        </p>
+                        <p class="note">
+                          Which FMTP parameters make up the media format is codec specific. In some cases
+                          a parameter can be omitted and still be inferred, in which case it is also a part
+                          of the media format of that codec.
+                        </p>
+                      </li>
+                      <li>
+                        <p>
+                          If <var>firstMediaFormat</var> is not equal to <var>secondMediaFormat</var>, return
+                          <code>false</code>.
+                        </p>
+                      </li>
+                      <li>
+                        <div id="codec-match-without-level-id">
+                        <p>
+                          If <var>ignoreLevels</var> is <code>false</code> and the highest complying bitstream
+                          levels inferred from <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}} and
+                          <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}} are different, return <code>false</code>.
+                        </p>
+                        <p class="note">
+                          Even if <var>ignoreLevels</var> is <code>true</code>, some codecs (such as H.264) include
+                          levels in the media format, so that the level cannot be ignored in this algorithm.
```

Comment:
More accurately "..., so that ignoring the level requires codec-specific parsing" to reflect level ID in H.264 being something that can be parsed out in some cases